src/d/actor/d_a_obj_barrel2.cpp

```diff
@@ -45,139 +47,34 @@ const dCcD_SrcCyl daObjBarrel2::Act_c::M_cyl_src = {
     },
     // cM3dGCylS
     {
-        /* Center */ 0.0f, 0.0f, 0.0f,
+        /* Center */ 0.0f,
+        0.0f,
+        0.0f,
```

Comment:
Looks like you ran an autoformatter on this whole file and now all of the data has weird linebreaks in it